src/main/java/net/mcreator/ui/dialogs/tools/WoodPackMakerTool.java

```diff
@@ -0,0 +1,447 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2020 Pylo and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.dialogs.tools;
+
+import net.mcreator.blockly.java.blocks.MCItemBlock;
+import net.mcreator.element.ModElementType;
+import net.mcreator.element.ModElementTypeRegistry;
+import net.mcreator.element.parts.MItemBlock;
+import net.mcreator.element.parts.Material;
+import net.mcreator.element.parts.StepSound;
+import net.mcreator.element.types.Block;
+import net.mcreator.element.types.Recipe;
+import net.mcreator.element.types.Tag;
+import net.mcreator.element.types.Tool;
+import net.mcreator.generator.GeneratorConfiguration;
+import net.mcreator.generator.GeneratorStats;
+import net.mcreator.io.FileIO;
+import net.mcreator.io.ResourcePointer;
+import net.mcreator.minecraft.ElementUtil;
+import net.mcreator.minecraft.RegistryNameFixer;
+import net.mcreator.ui.MCreator;
+import net.mcreator.ui.action.ActionRegistry;
+import net.mcreator.ui.action.BasicAction;
+import net.mcreator.ui.component.JColor;
+import net.mcreator.ui.component.util.PanelUtils;
+import net.mcreator.ui.dialogs.MCreatorDialog;
+import net.mcreator.ui.init.ImageMakerTexturesCache;
+import net.mcreator.ui.init.UIRES;
+import net.mcreator.ui.minecraft.MCItemHolder;
+import net.mcreator.ui.validation.Validator;
+import net.mcreator.ui.validation.component.VTextField;
+import net.mcreator.ui.validation.validators.ModElementNameValidator;
+import net.mcreator.util.ListUtils;
+import net.mcreator.util.StringUtils;
+import net.mcreator.util.image.ImageUtils;
+import net.mcreator.workspace.Workspace;
+import net.mcreator.workspace.elements.ModElement;
+
+import javax.swing.*;
+import java.awt.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Locale;
+
+public class WoodPackMakerTool {
+
+	private static void open(MCreator mcreator) {
+		MCreatorDialog dialog = new MCreatorDialog(mcreator, "Wood pack maker", true);
+		dialog.setLayout(new BorderLayout(10, 10));
+
+		dialog.setIconImage(UIRES.get("16px.toolpack").getImage());

```

Comment:
Make new icon, or use no icon.

Reply:
Don't worry, he said that he'll change the icon. :)

Reply:
My bad, did not notice :D I suggest adding placeholder than and fixing name, and then Matej will work on it ;)